docs/setup-guide/scale-out-local-dev-setup.md

```diff
@@ -57,7 +57,7 @@ Note:
 export SCALE_OUT_PROXY_IP=[PROXY_IP]
 export SCALE_OUT_PROXY_PORT=8888
 export IS_RESOURCE_PARTITION=true
-export TENANT_SERVERS=http://[TP1_IP]:8080,http://[TP2_IP]:8080
+export TENANT_SERVER=[TP1_IP]<,[TP2_IP]>
```

Comment:
should change to kubeconfigs

Reply:
Kubeconfig is generated from TENANT_SERVER automatically. Otherwise, dev needs to provide their own kubeconfig in local cluster up. This is not ideal.